homeassistant/components/atag/water_heater.py

```diff
@@ -0,0 +1,79 @@
+"""ATAG water heater component."""
+from homeassistant.components.water_heater import (
+    ATTR_TEMPERATURE,
+    STATE_ECO,
+    STATE_PERFORMANCE,
+    WaterHeaterDevice,
+)
+from homeassistant.const import STATE_OFF, TEMP_CELSIUS
+
+from . import DOMAIN, AtagEntity
+
+SUPPORT_FLAGS_HEATER = 0
+OPERATION_LIST = [STATE_OFF, STATE_ECO, STATE_PERFORMANCE]
+
+
+async def async_setup_platform(hass, config, async_add_devices, _discovery_info=None):
+    """Atag updated to use config entry."""
+    pass
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Initialize DHW device from config entry."""
+    atag = hass.data[DOMAIN][config_entry.entry_id]
+    async_add_entities([AtagWaterHeater(atag, "DHW")])
```

Comment:
Declare a constant here as well e.g. see "Thermostat" above.

Reply:
included as dict in ENTITY_TYPES in this update, hope this is better?